### DISCUSSION TITLE
Options picker fields

### DISCUSSION THREAD

**Discussion by Bitello** [Budibase/budibase#9797] [Help] [answered]
Hello!

I use a table in BudiDB that, briefly, has Text (Nome), Number (Exercicio) and Long Form Text (Objeto) fields.

![image](https://user-images.githubusercontent.com/22456825/221180723-347748fd-ec27-4791-9ac2-be0ed1d56715.png)
 
![image](https://user-images.githubusercontent.com/22456825/221180804-e6dc9489-f0e9-4438-9f48-d054db609dae.png)

In the option field of the Options Picker, only the Nome column option appears.

![image](https://user-images.githubusercontent.com/22456825/221180831-d9d3e158-1f3e-4c37-bc23-ea52bfaf7f21.png)

Is it some limitation?

**Comment by chadgriff** [comment]
That was confusing to me as well. However, you would just enter the Field Name. The dropdown is a just a list of previously entered options.,

**Comment by andz-bb** [comment, marked as answer]
hey @Bitello, yes this is a limitation of the Options Picker component. At the moment it only supports standard text type fields.
